src/App.css

```diff
@@ -1,4 +1,5 @@
-html {
+html,
+body {
   background-color: #d0edf5;
```

Comment:
Non-blocker: we should update this to use mantine's [global themes](https://v1.mantine.dev/guides/dark-theme/) in the future

Reply:
Okay cool, probably will make it easier for dark mode